Sources/LeafKit/LeafLexer/LeafParameterTypes.swift

```diff
@@ -50,30 +50,111 @@ public enum Parameter: Equatable, CustomStringConvertible {
 /// `Keyword`s are identifiers which take precedence over syntax/variable names - may potentially have
 /// representable state themselves as value when used with operators (eg, `true`, `false` when
 /// used with logical operators, `nil` when used with equality operators, and so forth)
-public enum Keyword: String, Equatable {
-    case `in`, `true`, `false`, `self`, `nil`, `yes`, `no`
-}
+public enum LeafKeyword: String, Equatable {
+    // MARK: Public - Cases
+    
+    //               Eval -> Bool / Other
+    //            -----------------------
+    case `in`,    //
+         `true`,  //   X       T
+         `false`, //   X       F
+         `self`,  //   X             X
+         `nil`,   //   X       F     X
+         `yes`,   //   X       T
+         `no`     //   X       F
```

Comment:
General commentary on the keywords and their potential meaning as value types in context